src/__tests__/lib/containers/MarkdownSynapse.test.tsx

```diff
@@ -112,13 +115,63 @@ describe('it performs all functionality', () => {
     })
   })
 
+  describe('it renders a video widget', () => {
+    it.only('renders a video with given a synapseID', async () => {
+      const mockGetEntityWiki = jest.fn().mockResolvedValue({
+        markdown: '${youtube?videoId=Bey4XXJAqS8}',
+        // markdown: '${video?mp4SynapseId=syn21714374}'
+      })
+      SynapseClient.getEntityWiki = mockGetEntityWiki
+      const props: MarkdownSynapseProps = {
+        ownerId: '_',
+        token: '_',
+      }
+
+      const { wrapper } = await createShallowComponent(props)
+      expect(wrapper.find(SynapseVideo)).toHaveLength(1)
+    })
+
+    it.only('do not render a video without token', async () => {
+      const mockGetEntityWiki = jest.fn().mockResolvedValue({
+        markdown: '${video?mp4SynapseId=syn21714374}',
+      })
+      SynapseClient.getEntityWiki = mockGetEntityWiki
+      const props: MarkdownSynapseProps = {
+        ownerId: '_',
+        token: '_',
+      }
+      const { wrapper } = await createShallowComponent(props)
+      expect(wrapper.find(SynapseVideo).html()).toMatch(
+        '<div><div><p>You will need to sign in for access to that resource<p></div></div>',
+      )
+    })
+
+    it.only('renders a video with given height and width', async () => {
```

Comment:
small typo again- should be 'renders a video widget with a given height and width'